libpod/runtime.go

```diff
@@ -196,14 +204,17 @@ var (
 		StorageConfig:         storage.StoreOptions{},
 		ImageDefaultTransport: DefaultTransport,
 		StateType:             BoltDBStateStore,
-		RuntimePath: []string{
-			"/usr/bin/runc",
-			"/usr/sbin/runc",
-			"/usr/local/bin/runc",
-			"/usr/local/sbin/runc",
-			"/sbin/runc",
-			"/bin/runc",
-			"/usr/lib/cri-o-runc/sbin/runc",
+		OCIRuntime:            "runc",
+		OCIRuntimes: map[string][]string{
+			"runc": []string{
+				"/usr/bin/runc",
+				"/usr/sbin/runc",
+				"/usr/local/bin/runc",
+				"/usr/local/sbin/runc",
+				"/sbin/runc",
+				"/bin/runc",
+				"/usr/lib/cri-o-runc/sbin/runc",
+			},
```

Comment:
Anyway to use the values from libpod.conf here?

Reply:
how?  This is used only if libpod.conf doesn't override the default specified here